### PR TITLE
Add support for zef ecosystem groups

### PR DESCRIPTION
**Pull request by lizmat**
For more information, see:
  https://deathbykeystroke.com/articles/20211220-fez-orgs-a-winter-solstice-miracle.html

### CODE REVIEW

**Comment by skaji** [comment]
I cannot run rakudo on my macOS 12 right now.
Once rakudo 2021.12 is released, I will review this PR.

**Comment by lizmat** [comment]
FWIW, the MacOS Monterey / libuv issue has been worked around on HEAD.

But if you want to wait for 2021.12, that's ok with me as well.  I have a locally patched version to upload raku-community-modules using App::Mi6, see e.g. https://raku.land/zef:raku-community-modules/Slang::Roman

File: lib/App/Mi6.rakumod [hunk 1]
```diff
@@ -54,13 +54,25 @@ method !zef-user() {
...
 
+method !zef-groups() {
+    try require ::("App::Mi6::Fez");
+    if ::("App::Mi6::Fez") ~~ Failure {
+        die "To create a distribution for Zef ecosystem, you need to install fez first";
+    }
+    if ::("App::Mi6::Fez").groups -> @groups {
+        return @groups;
+    } else {
+        die "To create a distribution for Zef ecosystem, you need to execute 'fez register' and 'fez login' first.";
```
Comment (skaji) [inline lib/App/Mi6.rakumod:72]: I'm not belong to any group right now.
Then `::("App::Mi6::Fez").groups -> @groups` is falsy, so I get this `die`.

**Comment by lizmat** [comment]
Closing in favour of #140

**Pull request closed without merging**